mvr/runtime/mangata-polkadot/src/lib.rs

```diff
@@ -365,9 +381,14 @@ impl parachain_info::Config for Runtime {}
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	pub const FixedReserveAssetTransferTrapCost : Balance = 10_000_000_000; // 0.01 DOT
+}
+
 impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type XcmExecutor = XcmExecutor<XcmConfig>;
+	type XcmExecutor =
+		xcm_config::ExecutorWrapper<XcmExecutor<XcmConfig>, FixedReserveAssetTransferTrapCost>;
```

Comment:
sooo, the wrapper has to be used only here right,  the other pallets in xcm_config can use base XcmExecutor ?

Reply:
exactly